src/generators.mjs

```diff
@@ -51,6 +52,16 @@ const createGenerator = markdownInput => {
    * @param {import('./generators/types.d.ts').GeneratorOptions} options The options for the generator runtime
    */
   const runGenerators = async ({ generators, ...extra }) => {
+    try {
```

Comment:
I do believe at the very least this code should live somewhere else, and not here.